script.js

```diff
@@ -295,27 +325,49 @@ async function next() {
     'page_path': '/next/' + state.lang.speechTag,
   });
   try {
-    console.info('Finding article.');
-    let article = await getArticleForLocation();
-    if (!article) {
-      console.info('Did not find location article. Trying nearby.');
-      article = await getNearbyArticle();
-    }
     if (!article) {
-      console.info('Did not find article');
-      setTimeout(next, pollSeconds * 1000);
-      return;
+      try {
+        console.info('Finding article.');
+        article = await getArticleForLocation()
+
+      } catch(e) {
```

Comment:
wrap `getArticleForLocation` in a `try/catch` to properly handle any error thrown in the function, so `next` can continue to use other strategies to find articles.